index.html

```diff
@@ -15,5 +15,9 @@ <h1>Titre</h1>
 </body>
   <script>
     document.queryString('h1').forEach(elt => elt.style.display = 'none');
+
+    setInterval(()=>{
```

Comment:
intéressant mais que penses-tu d'un timeout
```suggestion
    setTimeout(()=>{
```